templates/paragraphs/paragraph--accordion.html.twig

```diff
@@ -0,0 +1,9 @@
+{{ attach_library('atomic/accordion') }}
+
+{% embed "@molecules/accordion/accordion.twig" with {
+  accordion__hading: content.field_heading,
+} %}
+  {% block accordion__items %}
+    {{ content.field_accordion_item }}
```

Comment:
This is valid and works. I asked a similar question in a different PR around this pattern. I'm questioning if it makes sense to pass an entire drupal field to the component library. This feels overly opinionated. I wonder if things should loop in the Drupal template and then wire to something with a more predictable shape. Just mulling it over.

Reply:
Same as before. This is a standard pattern. The component library component supports nested components/multi-value fields like this with Twig blocks.